scripts/us_census/acs5yr/subject_tables/subject_table_test.py

```diff
@@ -30,8 +30,8 @@
 
 # These directories are excluded from testing
 _EXCLUDE_DIRS = [
-    'common', 'S0701', 'S0701PR', 's2201', 's0902', 's1201', 'S1810', 's2201',
-    'S2601A', 'S2601APR', 'S2602', 'S2602PR', 'S2603', '__pycache__', '~'
+    'common', 'S0701', 'S0701PR', 's2201', 's1201', 'S1810', 's2201', 'S2601A',
```

Comment:
s2201 is repeated

Reply:
Thanks for pointing this out, I removed the repeated element

Reply:
This line has more changes in master. Can you please merge again?

Reply:
Done! I have synced the changes from master and have resolved the merge conflicts